lib/mllp/client.ex

```diff
@@ -34,6 +40,67 @@ defmodule MLLP.ClientContract do
 end
 
 defmodule MLLP.Client do
+  @moduledoc """
+  MLLP.Client provides a simple tcp client for sending and receiving data i
```

Comment:
Should this be just sending data? also there is letter "i" doing something alone

Reply:
vim derp :)